### PR TITLE
[4.0] Remove nowrap

### PR DESCRIPTION
**Pull request by brianteeman**
This PR removes the class "nowrap" from the markup in the admin as it doesn't exist in the css and where it is needed it is already part of an existing class

NOTE: This requires #21938 and #21967 to be merged first

### CODE REVIEW

File: administrator/components/com_templates/tmpl/styles/default.php [hunk 1]
```diff
@@ -39,24 +39,24 @@
 					<table class="table" id="styleList">
 						<thead>
 							<tr>
-								<td style="width:1%" class="nowrap text-center">
+								<td style="width:1%" class="text-center">
```
Comment (Quy) [inline administrator/components/com_templates/tmpl/styles/default.php:42]: Remove `class="text-center"`. Not necessary for a non-breaking space.

File: administrator/components/com_tags/tmpl/tags/default.php [hunk 2]
```diff
@@ -71,49 +71,49 @@
 			<table class="table" id="categoryList">
 				<thead>
 					<tr>
-						<th scope="col" style="width:1%" class="nowrap d-none d-md-table-cell center">
+						<th scope="col" style="width:1%" class="d-none d-md-table-cell center">
```
Comment (Quy) [inline administrator/components/com_tags/tmpl/tags/default.php:74]: `text-center`?

File: administrator/components/com_menus/tmpl/items/default.php [hunk 3]
```diff
@@ -58,43 +58,43 @@
...
 							<?php endif; ?>
 							<?php if ($assoc) : ?>
-								<th scope="col" style="width:10%" class="nowrap d-none d-md-table-cell text-center">
+								<th scope="col" style="width:10%" class="d-none d-md-table-cell text-center">
 									<?php echo HTMLHelper::_('searchtools.sort', 'COM_MENUS_HEADING_ASSOCIATION', 'association', $listDirn, $listOrder); ?>
 								</th>
 							<?php endif; ?>
 							<?php if (($this->state->get('filter.client_id') == 0) && (Multilanguage::isEnabled())) : ?>
-								<th scope="col" style="width:10%" class="nowrap d-none d-md-table-cell text-center">
+								<th scope="col" style="width:10%" class="d-none d-md-table-cell text-center">
```
Comment (Quy) [inline administrator/components/com_menus/tmpl/items/default.php:93]: Remove `text-center`?

File: administrator/components/com_languages/tmpl/languages/default.php [hunk 4]
```diff
@@ -45,37 +45,37 @@
 					<table class="table" id="contentList">
 						<thead>
 							<tr>
-								<th scope="col" style="width:1%" class="nowrap text-center d-none d-md-table-cell">
+								<th scope="col" style="width:1%" class="text-center d-none d-md-table-cell">
 									<?php echo HTMLHelper::_('searchtools.sort', '', 'a.ordering', $listDirn, $listOrder, null, 'asc', 'JGRID_HEADING_ORDERING', 'icon-menu-2'); ?>
 								</th>
-								<td style="width:1%"  class="nowrap text-center">
+								<td style="width:1%"  class="text-center">
```
Comment (Quy) [inline administrator/components/com_languages/tmpl/languages/default.php]: Remove extra space before `class`

File: administrator/components/com_finder/tmpl/index/default.php [hunk 5]
```diff
@@ -33,25 +33,25 @@
...
-							<th scope="col" style="width:10%" class="nowrap d-none d-md-table-cell text-center">
+							<th scope="col" style="width:10%" class="d-none d-md-table-cell text-center">
 								<?php echo JHtml::_('searchtools.sort', 'COM_FINDER_INDEX_HEADING_INDEX_TYPE', 't.title', $listDirn, $listOrder); ?>
 							</th>
-							<th scope="col" style="width:10%" class="nowrap d-none d-md-table-cell text-center">
+							<th scope="col" style="width:10%" class="d-none d-md-table-cell text-center">
 								<?php echo JHtml::_('searchtools.sort', 'COM_FINDER_INDEX_HEADING_INDEX_DATE', 'l.indexdate', $listDirn, $listOrder); ?>
 							</th>
-							<th scope="col" style="width:15%" class="nowrap text-center d-none d-md-table-cell text-center">
+							<th scope="col" style="width:15%" class="text-center d-none d-md-table-cell text-center">
```
Comment (Quy) [inline administrator/components/com_finder/tmpl/index/default.php]: Remove duplicate `text-center`

File: administrator/components/com_associations/tmpl/associations/modal.php [hunk 6]
```diff
@@ -60,30 +60,30 @@
 			<thead>
 				<tr>
 					<?php if (!empty($this->typeSupports['state'])) : ?>
-						<th scope="col" style="width:1%" class="center nowrap">
+						<th scope="col" style="width:1%" class="center">
```
Comment (Quy) [inline administrator/components/com_associations/tmpl/associations/modal.php:63]: Change to `text-center`

File: administrator/components/com_templates/tmpl/styles/default.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_templates/tmpl/styles/default.php:42]: beyond the scope of this PR but if you remove it you will see it is needed

File: administrator/components/com_tags/tmpl/tags/default.php (hunk 2 above)
Comment (brianteeman) [reply administrator/components/com_tags/tmpl/tags/default.php:74]: beyond the scope of this pr

File: administrator/components/com_menus/tmpl/items/default.php (hunk 3 above)
Comment (brianteeman) [reply administrator/components/com_menus/tmpl/items/default.php:93]: beyond the scope of this PR and iirc already addressed in another pr of mine

File: administrator/components/com_associations/tmpl/associations/modal.php (hunk 6 above)
Comment (brianteeman) [reply administrator/components/com_associations/tmpl/associations/modal.php:63]: beyond the scope of this pr

File: administrator/components/com_finder/tmpl/index/default.php (hunk 5 above)
Comment (brianteeman) [reply administrator/components/com_finder/tmpl/index/default.php]: done

File: administrator/components/com_languages/tmpl/languages/default.php (hunk 4 above)
Comment (brianteeman) [reply administrator/components/com_languages/tmpl/languages/default.php]: done

**Comment by ceus1984** [comment]
And what are the test-instructions. I have taken part in several PBFs but then there were always exact test instructions given, what I am missing in several cases today

**Comment by brianteeman** [comment]
closing for now

**Comment by brianteeman** [comment]
going to have to redo this one to fix the conflicts

**Comment by brianteeman** [comment]
see #22073

**Pull request closed without merging**